### DISCUSSION TITLE
Hello packet doesn't arrived to neighbor

### DISCUSSION THREAD

**Discussion by JYBESiU** [ANLAB-KAIST/KENSv3#254] [Project Q&A 3. Internet Layer (IP/Routing)]
We send the packet with the exactly same values in #249 and but the packet doesn't arrived to the neighbor.
We setup the timer in HelloInt second.
I found that my ip from getIPAddr(0) is same with the source IP in the arrived packet.

I think 10.0.1.1 should get packet from 10.0.1.2. Why this packet come back to origin?

<img width="1391" alt="스크린샷 2024-05-28 오후 4 30 32" src="https://github.com/ANLAB-KAIST/KENSv3/assets/39625161/d966c2cd-deb3-4cc1-aa80-b1c1e4533aa1">

**Comment by rs0909** [comment]
#246 Solved!
